front-end/src/components/Calendar/CalendarBody/index.jsx

```diff
@@ -70,6 +75,19 @@ const CalendarBody = (props) => {
   const plusIconClickHandler = (index) => {
     const newNumberOfCells = [...events];
     newNumberOfCells[index].push({});
+    AxiosInstance.post("/events/setEventGroup", {
+      eventType: variant,
+      eventDate: parseInt(newNumberOfCells[index][0].event_date),
+      slots:  newNumberOfCells[index].length
+    });
```

Comment:
should we catch errors on these requests and log

Reply:
I'll have Cem handle that with the UI